### PR TITLE
WWST-2004 Update Stelpro Maestro DTH with deviceJoinNames from mfg

### PR DESCRIPTION
**Pull request by dkirker**
Update for Stelpro Orleans Fan-Heater and Stelpro Sonoma Fan-Heater

### CODE REVIEW

**Comment by dkirker** [comment]
cc @stelpro

File: devicetypes/stelpro/stelpro-maestro-thermostat.src/stelpro-maestro-thermostat.groovy [hunk 1]
```diff
@@ -44,8 +44,8 @@ metadata {
 		command "updateWeather"
 
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0201, 0204, 0405", outClusters: "0003, 000A, 0402", manufacturer: "Stelpro", model: "MaestroStat", deviceJoinName: "Stelpro Maestro Thermostat"
-		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0201, 0204, 0405", outClusters: "0003, 000A, 0402", manufacturer: "Stelpro", model: "SORB", deviceJoinName: "Stelpro Maestro Thermostat"
-		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0201, 0204, 0405", outClusters: "0003, 000A, 0402", manufacturer: "Stelpro", model: "SonomaStyle", deviceJoinName: "Stelpro Maestro Thermostat"
+		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0201, 0204, 0405", outClusters: "0003, 000A, 0402", manufacturer: "Stelpro", model: "SORB", deviceJoinName: "Stelpro Orleans Fan-Heater"
```
Comment (tpmanley) [inline devicetypes/stelpro/stelpro-maestro-thermostat.src/stelpro-maestro-thermostat.groovy]: Is there supposed to be a dash between Fan and Heater? The website doesn't have one: https://www.stelpro.com/en-US/sonoma-style-wall-fan-heater

Comment (dkirker) [reply devicetypes/stelpro/stelpro-maestro-thermostat.src/stelpro-maestro-thermostat.groovy]: Those are the values that I received from @stelpro . Also, realized that I had left out "Style" from the Sonoma string, so I just updated that.

**Review by tpmanley — DISMISSED** [review]

**Comment by tpmanley** [comment]
Will wait to merge until tomorrow in case @stelpro wants to review too

**Comment by stelpro** [comment]
Hi,
Tom Manley is right.
There should not be a dash between Fan and Heater.
My bad.

**Comment by dkirker** [comment]
Cool, I'll update this PR.

@stelpro  I noticed the marketing on the website has Orleans as ORLÉANS, we can use that in the device join name if you'd prefer.

**Comment by stelpro** [comment]
Hi Donald,

Yes indeed, if the accent is not an issue, please use them.
Just make sure it won't break anything or show up in a weird way on the app!

**Comment by dkirker** [comment]
Updated! @stelpro @tpmanley  Let me know if those look good!

**Review by tpmanley — APPROVED** [review]

**Comment by stelpro** [comment]
All good!

**Comment by dkirker** [comment]
@tpmanley or @workingmonk Could you please merge this?

**Pull request merged**